### PR TITLE
【Hackathon 6th Fundable Projects 3 No.338】sequence_expand

### PR DESCRIPTION
**Pull request by co63oc**
### PR Category

Others 

### PR Types

Others 

### Description

338 sequence_expand

### CODE REVIEW

File: paddle/phi/kernels/impl/sequence_expand_kernel_impl.cu.h
Comment (zyfncg) [inline paddle/phi/kernels/impl/sequence_expand_kernel_impl.cu.h:1]: impl文件能否拆分到前反向kernel文件中？

File: paddle/phi/kernels/impl/sequence_expand_kernel_impl.h
Comment (zyfncg) [inline paddle/phi/kernels/impl/sequence_expand_kernel_impl.h:1]: 同上

File: paddle/phi/ops/yaml/ops.yaml
Comment (zyfncg) [inline paddle/phi/ops/yaml/ops.yaml:1]: image

sequence系列算子只支持静态图，加到静态图的yaml中吧

**Comment by co63oc** [comment]
冲突合入 https://github.com/PaddlePaddle/Paddle/pull/65588

**Pull request closed without merging**